bbb-common-web/src/main/java/org/bigbluebutton/api/ParamsProcessorUtil.java

```diff
@@ -618,6 +620,24 @@ public Meeting processCreateParams(Map<String, String> params) {
         	guestPolicy = params.get(ApiParams.GUEST_POLICY);
 		    }
 
+        String uploadExternalDescription = defaultUploadExternalDescription;
+        if (!StringUtils.isEmpty(params.get(ApiParams.UPLOAD_EXTERNAL_DESCRIPTION))) {
+            uploadExternalDescription = params.get(ApiParams.UPLOAD_EXTERNAL_DESCRIPTION);
+        }
+
+        String uploadExternalUrl = defaultUploadExternalUrl;
+        if (!StringUtils.isEmpty(params.get(ApiParams.UPLOAD_EXTERNAL_URL))) {
+            uploadExternalUrl = params.get(ApiParams.UPLOAD_EXTERNAL_URL);
+
+            boolean hasQuoteCharacters = (uploadExternalUrl.charAt(0) == '"' && uploadExternalUrl.charAt(uploadExternalUrl.length()-1) == '"') 
+                || (uploadExternalUrl.charAt(0) == '\'' && uploadExternalUrl.charAt(uploadExternalUrl.length()-1) == '\'')
+                || (uploadExternalUrl.charAt(0) == '`' && uploadExternalUrl.charAt(uploadExternalUrl.length()-1) == '`');

```

Comment:
Would it hurt to also use `\"` instead of `"` and same for ``` \` ``` instead of ``` ` ```

Reply:
Is there some reason why we need to remove quote only to this param?
I mean, it seems to be useful but all the other params don't have this kind of process.. and doing it now seems it doesn't follow the pattern.

I don't know if this case specifically needs it for some reason..

Reply:
maybe it should include an URL validation instead! (rejecting if there are quotes embracing the string)

Reply:
Thanks for the feedback, guys! 

I'm removing that part in [81fb8ae](https://github.com/bigbluebutton/bigbluebutton/pull/15251/commits/81fb8ae74a7249f033fa15a3799e5a178769cd17) as this is something that can be further discussed and handled in another issue/PR in the future :)